src/main/scala/chiseltest/exceptions.scala

```diff
@@ -7,7 +7,8 @@ class NotLiteralException(message: String) extends ChiselTestException(message)
 class LiteralTypeException(message: String) extends ChiselTestException(message)
 class UnpokeableException(message: String) extends ChiselTestException(message)
 class UnsupportedOperationException(message: String) extends ChiselTestException(message)
-class FailedExpectException(val message: String, val failedCodeStackDepth: Int) extends ChiselTestException(message)
+class FailedExpectException(val message: String, val stackTraceElement: StackTraceElement)
```

Comment:
Consider making another exception superclass that is ChiselTestFailedException or something, to distinguish a test failure from a bad test or internal error? And then when you propagate failures to ScalaTest you can match on a more general superclass?